### PR TITLE
fix(encryptionKey): align dim and issuerComponent config

### PR DESCRIPTION
**Pull request by Phil91**
## Description

Align the credential and wallet configuration to use the same encryption key

## Why

The same key is used for encrypting and decryption the wallet data by different processes.

## Issue

Refs: #358

## Checklist

- [x] I have performed a self-review of my changes
- [x] I have successfully tested my changes
- [x] I have added comments in the default values.yaml file with helm-docs syntax ('# -- ') if relevant for installation
- [x] I have commented my changes, particularly in hard-to-understand areas

### CODE REVIEW

**Review by evegufy — CHANGES_REQUESTED** [review]
shouldn't it be the other way around as the issuerComponent encryption key must match the dim encryption key: I mean shouldn't the dim encryption key remain, as that is the key which does and will continue doing encrypting?

**Comment by Phil91** [comment]
> shouldn't it be the other way around as the issuerComponent encryption key must match the dim encryption key: I mean shouldn't the dim encryption key remain, as that is the key which does and will continue doing encrypting?

adjusted

File: environments/helm-values/values-int.yaml [hunk 1]
```diff
@@ -263,13 +263,10 @@ backend:
       universalResolverAddress: "https://dev.uniresolver.io/"
       encryptionConfigs:
         index0:
-          encryptionKey: "<path:portal/data/int/encryption-keys#dim-encryption-key0>"
+          encryptionKey: "<path:portal/data/int/processes-worker#dim-encryption-key0>"
```
Comment (evegufy) [inline environments/helm-values/values-int.yaml:266]: for the association env the path needs be `<path:portal/data/int/encryption-keys#dim-encryption-key0>`

**Review by evegufy — CHANGES_REQUESTED** [review]

File: environments/helm-values/values-int.yaml (hunk 1 above)
Comment (Phil91) [reply environments/helm-values/values-int.yaml]: adjusted the paths in the environment files

**Review by ntruchsess — APPROVED** [review]

**Pull request merged**